string uuid
string name
geometry_msgs/Pose2D[] path
OperationAction[] operationActions
bool automaticPlanning
